misc/datetimeinsert.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from talon import Context, actions, Module
+
+mod = Module()
+apps = mod.apps
```

Comment:
this line it looks like you don't need

Reply:
@fidgetingbits good point, I updated the pull request